### ISSUE TITLE
Vary page title

### ISSUE THREAD

**Issue by Zack-83** [rdmorganiser/rdmo#528]
### Rationale / Begründung

The subpages of an RDMO instance have always the same title "rdmocatalog".

![image](https://user-images.githubusercontent.com/37865804/190388828-932bc952-eaf7-4481-9a21-93ffdc9e2a92.png)

### Affected 

Users

### Minimal functionality

Might it be possible to differenciate the titles?
E.g. "RDMO-Management-Options", "RDMO-Project13-View14", etc.

**Comment by jochenklar** [comment]
Yes, this is possible and not super hard to do, but a bit cumbersome since every view needs to provide a `title` to `render`. Then it can be implemented like (I did) here: https://github.com/ISI-MIP/isimip-data/blob/master/isimip_data/core/templates/core/base.html#L6

**Comment by MyPyDavid** [comment]
could also be implemented with the JavaScript `document.title` or? when the values for `pk` are already there

**Comment by jochenklar** [comment]
No, then we would need JavaScript on every page, which we do not need right now. If we watn to change the title in the e.g. interview dynamically, then we need to use this.
